.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: ci
+on: [push, pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16'
+    - run: go get github.com/mattn/goveralls
+    - run: |
+        make test
+        make
+        make build.docker
+    - run: goveralls -coverprofile=profile.cov -service=github
+      env:
+        COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Push the latest Docker image
+      run: |
+        echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
```

Comment:
Is that how you're supposed to do this? Won't it echo the command and the secret?

Reply:
Also, does this mean that `DOCKERHUB_PASSWORD` is available to all the other steps in the action (like `goveralls`)?

Reply:
Good points, I have to double check that it does indeed prevent access from the other steps and is not visible anywhere. Will comment out this whole step for now.